src/App.tsx

```diff
@@ -141,11 +159,19 @@ function App() {
           }
         });
       } catch (error) {
-        setisExternallyConnectableConnected(false);
+        setIsExternallyConnectableConnected(false);
       }
     }
   };
 
+  const handleWalletListClick = useCallback(
+    (_extensionId: string) => {
```

Comment:
Can we rename this to something like `newExtensionId` maybe?

Reply:
resolved in [de81e](https://github.com/MetaMask/test-dapp-multichain/pull/22/commits/de81e7e392022e41ac8edd4066a44826540fb4d5)